### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory: '../data/ednet\\train_valid_sequences.csv'

### ISSUE THREAD

**Issue by Xemorr** [pykt-team/pykt-toolkit#146]
FileNotFoundError: [Errno 2] No such file or directory: '../data/ednet\\train_valid_sequences.csv'
when running wandb_saint_train.py --dataset_name=ednet

I have already ran python data_preprocess.py --dataset_name=ednet

**Comment by BojunZhan** [comment]
Thank you for your support of our work, have you changed the relevant code? I just tried the code on my side and it is working successfully. "... /data/ednet\\\train_valid_sequences.csv", the path where the file is loaded should be "... /data/ednet/train_valid_sequences.csv".

**Comment by Xemorr** [comment]
> Thank you for your support of our work, have you changed the relevant code? I just tried the code on my side and it is working successfully. "... /data/ednet\\train_valid_sequences.csv", the path where the file is loaded should be "... /data/ednet/train_valid_sequences.csv".

Yes, the file isn't in the location /data/ednet/train_valid_sequences. data_preprocess is only generating 
![image](https://github.com/pykt-team/pykt-toolkit/assets/31805746/ea337536-4586-4e0e-9377-338e31f6fdca)

I believe the weird directory notation is due to me being on Windows, but I don't believe it is affecting the functionality as train_valid_sequences was not generated by data_preprocess.

**Comment by sonyawong** [comment]
> > Thank you for your support of our work, have you changed the relevant code? I just tried the code on my side and it is working successfully. "... /data/ednet\train_valid_sequences.csv", the path where the file is loaded should be "... /data/ednet/train_valid_sequences.csv".
> 
> Yes, the file isn't in the location /data/ednet/train_valid_sequences. data_preprocess is only generating ![image](https://private-user-images.githubusercontent.com/31805746/286602640-ea337536-4586-4e0e-9377-338e31f6fdca.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE5NjY2MzQsIm5iZiI6MTcwMTk2NjMzNCwicGF0aCI6Ii8zMTgwNTc0Ni8yODY2MDI2NDAtZWEzMzc1MzYtNDU4Ni00ZTBlLTkzNzctMzM4ZTMxZjZmZGNhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjA3VDE2MjUzNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE1ZjE5MjQxY2VmOGM3NmE3YjkxY2ZjMzdjNjJkMmE0NzhhMWI3YTE1ZTgzMTI3Njg5ZDM4YjcyYzYyZjU2MzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.y1wCCiCDnCd1oko73ZKl50JMRI67SB5l2Fo9BANhhwM)
> 
> I believe the weird directory notation is due to me being on Windows, but I don't believe it is affecting the functionality as train_valid_sequences was not generated by data_preprocess.

Hi, when u conduct the data preprocess by command "python data_preprocess.py", you will get the train_valid_sequences.csv after running successfully. And our default raw data are located in the data directory. Please make sure your data is in the "data/ednet/". Thank you.
